File: Lab/Week1/score.c
#include <stdio.h>

int main(void){
    char name[20];
    int score[3] = {0, };
    int sum = 0;
    char title[][20] = {"국어", "수학", "영어"};
    
    printf("학생 이름 입력: ");
    scanf("%s", name);
    for(int i=0; i<3; i++){
        printf("%s 학생의 %s 점수 입력: ", name, title[i]);
        scanf("%d", &score[i]);
        sum += score[i];
    }
    
    float avg = sum / 3.0;
    printf("%s 학생의 점수 합계는 %d이고, 평균은 %.1f입니다.\n", name, sum, avg);
    return 0;
}

File: Lab/Java/Lab09/J083/Score.java
public class Score {
    private String name;
    private int kor, eng, mat;
    private int sum;
    private double avg;

    public Score(String name, int kor, int eng, int mat) {
        this.name = name;
        this.kor = kor;
        this.eng = eng;
        this.mat = mat;
        this.sum = kor + eng + mat;
        this.avg = sum / 3.0;
    }

    public int getKor() { return kor; }
    public int getEng() { return eng; }
    public int getMat() { return mat; }
    public int getSum() { return sum; }
    public double getAvg() { return avg; }
    public String getName() { return name; }

    @Override
    public String toString() {
        return name + " - Total = " + sum + ", Average = " + String.format("%.2f", avg);
    }
}
